src/helpers/Utils.js

```
@@ -186,3 +187,69 @@ export const availableDataVisibilityFilter = (data) => {
+    "Thick_Ascending_Limb_Of_Loop_Of_Henle": CellTypeEnum.THICK_ASCENDING_LIMB_LOOP_OF_HENLE,
+};
+
+export const cellMapToOntologyId = {
+    "Glomerulus / Renal Corpuscle": "", 
+    "Glomerular Parietal Epithelium": "CL_1000452",
+    "Parietal Epithelial Cell": "CL_1000452",
+    "Glomerular Mesangium": "CL_1000742",
+    "Mesangial Cell": "CL_1000742",
+    "Glomerular Capillary Endothelium": "CL_1001005",
+    "Glomerular Capillary Endothelial Cell": "CL_1001005",
+    "Proximal Tubule": "CL_0002306",
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)